src/main/java/pl/winiarczyk/lesson4/homework/MinToSecConversion/MinutesToSecondsConversion.java

```diff
@@ -0,0 +1,17 @@
+package pl.winiarczyk.lesson4.homework.MinToSecConversion;
+
+class MinutesToSecondsConversion {
+
+    double timeInMinutes;
+
+    public MinutesToSecondsConversion(double minutes) {
+        this.timeInMinutes = timeInMinutes;
+    }
+
+    public static double convertMinutesToSeconds(double timeInMinutes) {
+
+        double timeInSeconds;
+        return timeInSeconds = timeInMinutes * 60;
+
+    }
```

Comment:
Po co nam tutaj wszystko to co zaznaczyłem kolorkiem? ;)

![image](https://user-images.githubusercontent.com/17989854/131352001-deed3ffd-a818-41a4-91f8-fbf6645d1da7.png)